ide/docker.api/src/org/netbeans/modules/docker/api/DockerAction.java

```diff
@@ -1004,9 +1004,8 @@ public StreamItem fetch() {
         }
     }
 
-    public boolean ping() {
+    public boolean pingWithExceptions() throws IOException, MalformedURLException {
```

Comment:
I would not use such a precise `throws` clause. I would use `Exception` that way the code can evolve without being tied to the fixed API.